docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -5,6 +5,7 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 [id="doc-contribute-howto"]
 = Contribute to Quarkus documentation
 include::_attributes.adoc[]
+:diataxis-type: howto
```

Comment:
Line 46-58

It would be good to update the example header code snippet here too to include the new content type attribute. 

For example (I reused what you added to doc-reference.adoc.]

```
[source,asciidoc]
----
[id="security-basic-authentication-howto"] <1>
= Secure a Quarkus application with basic authentication <2>
include::_attributes.adoc[] <3>
:diataxis-type: howto <4>
:categories: security,web <5>
<6>
----
<1> Set the `id` value to be the same as the file name but without the extension. You can shorten this if the file name is too long.
<2> For information about how to create a good title for each content type, see xref:{doc-guides}/doc-reference.adoc#titles-and-headings[Titles and headings] on the "Quarkus style and content guidelines" page.
<3> The `_attributes.adoc` include is required to ensure that attributes get resolved and the table of contents is generated.
<4> Specify the diataxis content type: `concept`, `howto`, `reference`, or `tutorial`.
<5> Set at least one category to ensure that the content is findable on the link:https://quarkus.io/guides/[Quarkus documentation home page]. For a list of Quarkus categories, see xref:{doc-guides}/doc-reference.adoc#document-attributes-and-variables[Document attributes and variables] on the "Quarkus style and content guidelines" page.
<6> Insert a blank line before the abstract.
```